src/webapp/pages/Router.tsx

```diff
@@ -1,19 +1,42 @@
 import React from "react";
 import { HashRouter, Route, Switch } from "react-router-dom";
-import { ExamplePage } from "./example/ExamplePage";
-import { LandingPage } from "./landing/LandingPage";
+
+import { DashboardPage } from "./dashboard/DashboardPage";
+import { EventTrackerPage } from "./event-tracker/EventTrackerPage";
+import { IncidentActionPlanPage } from "./incident-action-plan/IncidentActionPlanPage";
+import { ResourcesPage } from "./resources/ResourcesPage";
+import { IMTeamBuilderPage } from "./incident-management-team-builder/IMTeamBuilderPage";
+import { CreateEventPage } from "./create-event/CreateEventPage";
+import { CreateRiskAssessmentPage } from "./create-risk-assessment/CreateRiskAssessmentPage";
+import { CreateIncidentActionPlanPage } from "./create-incident-action-plan/CreateIncidentActionPlanPage";
+import { AssignRolePage } from "./assign-role/AssignRolePage";
 
 export function Router() {
     return (
         <HashRouter>
             <Switch>
+                <Route path="/create-event" render={() => <CreateEventPage />} />
+                <Route path="/event-tracker/:event" render={() => <EventTrackerPage />} />
                 <Route
-                    path="/for/:name?"
-                    render={({ match }) => <ExamplePage name={match.params.name ?? "Stranger"} />}
+                    path="/create-risk-assessment/:event"
+                    render={() => <CreateRiskAssessmentPage />}
                 />
-
+                <Route
+                    path="/incident-management-team-builder/:event"
```

Comment:
Now that I have a clearer idea of how to make the form and that we are going to have a configuration to inject it including title, subtitle... And that all forms has same UI structure, I think this could be done and it is better for the idea of making it more generic :tada:

Reply:
Typically, we want different URLs for each page of the app. Then, internally, we are free to choose the implementation for these pages/forms/... (with a bottom-up of top-down flavor, to avoid duplication)

Reply:
I have improved the paths in Router, now each form created or edited will have a different url